From e02e4096be38bc472c6823997686d8a7b34b3396 Mon Sep 17 00:00:00 2001
From: Tomas Glozar <tglozar@gmail.com>
Date: Tue, 31 Oct 2023 19:51:09 +0100
Subject: [PATCH 07/70] ia64: Remove unused syscall_lookup_dcookie

lookup_dcookie was removed on other architectures after ia64 was
dropped.
---
 arch/ia64/kernel/syscalls/syscall.tbl | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/ia64/kernel/syscalls/syscall.tbl b/arch/ia64/kernel/syscalls/syscall.tbl
index 83d8609aec03..ee4311d08593 100644
--- a/arch/ia64/kernel/syscalls/syscall.tbl
+++ b/arch/ia64/kernel/syscalls/syscall.tbl
@@ -222,7 +222,7 @@
 210	common	fadvise64			sys_fadvise64_64
 211	common	tgkill				sys_tgkill
 212	common	exit_group			sys_exit_group
-213	common	lookup_dcookie			sys_lookup_dcookie
+213	common	lookup_dcookie			sys_ni_syscall
 214	common	io_setup			sys_io_setup
 215	common	io_destroy			sys_io_destroy
 216	common	io_getevents			sys_io_getevents
-- 
2.25.1

